.env.production

```diff
@@ -1,5 +1,4 @@
 API_BASE_URL=https://api.missionhub.com
-GCM_SENDER_ID=208966923006
```

Comment:
What was this used for exactly?

Reply:
I think an old version needed it. Maybe GCM before it migrated to FCM. My comment above says it's "pulled from native firebase config" now so my old self must have read something about that... It's missing from the new typescript types and the docs now.